CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.1.9 (2022-01-28)
+## iotedged
+### Bug Fixes
+* Fix moby check [dc6cee2](https://github.com/Azure/iotedge/commit/dc6cee24e6b7b9b5380e5a6f6e5da28344e0e33c)
+* Removing moby check for non-windows [b71b828](https://github.com/Azure/iotedge/commit/b71b828f0304dace8c2c1225e8c29eced9ab285c)
+
+
+## Edge Agent
+### Bug Fixes
+* Fix for concurrent module creation with workload socket [1a30568](https://github.com/Azure/iotedge/commit/1a30568e55790011919295e23938f5b1cef2dc14)
+* Update Base Images for a Security Patch [7b689b3](https://github.com/Azure/iotedge/commit/7b689b3d58f2b732f7ff5138ba36c9c331449322)
+
```

Comment:
`https://github.com/Azure/iotedge/commit/46f3f7e04cfe0beaebc6ea9764611765a47107fb` is missing.